peers-jws/src/main/java/net/sourceforge/peers/javawebstart/RegisterSIPClient.java

```diff
@@ -8,22 +8,25 @@
 import net.sourceforge.peers.sip.transport.SipRequest;
 import net.sourceforge.peers.sip.transport.SipResponse;
 
+import java.io.File;
+
 public class RegisterSIPClient {
     private EventManager eventManager;
     private Registration registration;
     private Logger logger;
-
+    public final static String CONFIG_FILE = "conf" + File.separator + "peers.xml";
     public RegisterSIPClient(final String[] args) {
         String peersHome = Utils.DEFAULT_PEERS_HOME;
         if (args.length > 0) {
             peersHome = args[0];
         }
         logger = new FileLogger(peersHome);
         registration = new Registration(logger);
+        AbstractSoundManager soundManager = new JavaxSoundManager(false, logger, peersHome);;
```

Comment:
Syntax - two `;`
Any reason it was changed, seems to be the same, except not final?